src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/DraftAdjudicationIntTest.kt

```diff
@@ -233,13 +135,6 @@ class DraftAdjudicationIntTest : IntegrationTestBase() {
       .expectStatus().isCreated
       .expectBody()
       .jsonPath("$.draftAdjudication.id").isNumber
-      .jsonPath("$.draftAdjudication.prisonerNumber").isEqualTo(testAdjudication.prisonerNumber)
-      .jsonPath("$.draftAdjudication.incidentDetails.dateTimeOfIncident")
-      .isEqualTo(testAdjudication.dateTimeOfIncidentISOString)
-      .jsonPath("$.draftAdjudication.incidentDetails.handoverDeadline")
-      .isEqualTo(testAdjudication.handoverDeadlineISOString)
-      .jsonPath("$.draftAdjudication.incidentDetails.locationId").isEqualTo(testAdjudication.locationId)
-      .jsonPath("$.draftAdjudication.incidentStatement.statement").isEqualTo("test")
```

Comment:
Now I look at it, I suppose it would have made sense to keep the checks for the actual thing that is been added/changed.

So, for example, a test to `Edit incident statement` should check the incident statement